src/coprocessor/endpoint.rs

```diff
@@ -275,7 +153,8 @@ impl Host {
                         tracker.record_handle(Some(&mut resp), exec_metrics);
                         future::ok::<_, ()>(on_resp.respond(resp))
                     };
-                    return pool.spawn_fn(do_request).forget();
+                    pool.spawn(do_request).forget();
```

Comment:
???

Reply:
previously, pool is a CpuPool, which signature is `spawn_fn`. Now pool is a FuturePool, which signature is `spawn`.